chore: update pr template docs (#18922)
renovatebot/renovate: .github/pull_request_template.md

<!-- Describe why you're making these changes if it's not already explained in a corresponding issue. -->

> <!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number -->
> <!-- If you're referencing and issue with this pull request, use a dash `-` at line start to see the issue title -->

Reviewer: ```suggestion
<!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number. -->
<!-- If you're referencing an issue with this pull request, put it in a Markdown list like this: - #issue_number. -->
```
